app/components/EventEditor/index.js

```diff
@@ -1,17 +1,227 @@
+        </div>
+        <div>
+          <label>Company</label>
+          <Field name={speaker + 'company'} type="text" component="input" />
+        </div>
+        <div>
+          <label>Talk Title</label>
+          <Field name={speaker + 'talkTitle'} type="text" component="input" />
+        </div>
+        <div>
+          <label>Talk Summary</label>
+          <Field name={speaker + 'talkSummary'} type="text" component="input" />
+        </div>
+        <div>
+          <label>Twitter Handle</label>
+          <Field name={speaker + 'twitterHandle'} type="text" component="input" />
+        </div>
+        <div>
+          <label>GitHub Handle</label>
+          <Field name={speaker + 'githubHandle'} type="text" component="input" />
+        </div>
+        <div>
+          <label>Blog URL</label>
+          <Field name={speaker + 'blogURL'} type="text" component="input" />
+        </div>
+      </li>
+    ))}
+  </ul>
+);
+
+Speakers.propTypes = {
+  fields: PropTypes.shape({
+    push: PropTypes.func,
+    remove: PropTypes.func,
+  }),
+};
+
+const Schedule = ({ fields }) => (
+  <ul>
+    <div>
+      <button type="button" onClick={() => fields.push()}>Add Schedule Item</button>
+    </div>
+    {fields.map((item, index) => (
+      <li key={index}>
+        <button type="button" onClick={() => fields.remove(index)} >
+          Remove Schedule Item
+        </button>
+        <div>
+          <label>Time</label>
+          <Field name={item + 'time'} type="text" component="input" />
+        </div>
+        <div>
+          <label>Text</label>
+          <Field name={item + 'text'} type="text" component="input" />
+        </div>
+      </li>
+    ))}
+  </ul>
+);
+
+Schedule.propTypes = {
+  fields: PropTypes.shape({
+    push: PropTypes.func,
+    remove: PropTypes.func,
+  }),
+};
+
+const Sponsors = ({ fields }) => (
+  <ul>
+    <div>
+      <button type="button" onClick={() => fields.push()}>Add Sponsor</button>
+    </div>
+    {fields.map((sponsor, index) => (
+      <li key={index}>
+        <button type="button" onClick={() => fields.remove(index)} >
+          Remove Sponsor
+        </button>
+        <div>
+          <label>Image</label>
+          <Field name={sponsor + 'imageURL'} type="text" component="input" />
+        </div>
+        <div>
+          <label>URL</label>
+          <Field name={sponsor + 'websiteURL'} type="text" component="input" />
+        </div>
+      </li>
+    ))}
+  </ul>
+);
+
+Sponsors.propTypes = {
+  fields: PropTypes.shape({
+    push: PropTypes.func,
+    remove: PropTypes.func,
+  }),
+};
+
+const UpcomingEvents = ({ fields }) => (
+  <ul>
+    <div>
+      <button type="button" onClick={() => fields.push()}>Add Upcoming Event</button>
+    </div>
+    {fields.map((event, index) => (
+      <li key={index}>
+        <button type="button" onClick={() => fields.remove(index)} >
+          Remove Event
+        </button>
+        <div>
+          <label>Title</label>
```

Comment:
please add the 'for' attribute to these labels, so they can be clicked on as per this example:
https://developer.mozilla.org/en/docs/Web/HTML/Element/label#Using_the_for_attribute